packages/core/src/Stores/client-store.js

```diff
@@ -2581,13 +2581,11 @@ export default class ClientStore extends BaseStore {
     }
 
     getChangeableFields() {
-        const get_settings =
-            Object.keys(this.account_settings).length === 0
-                ? WS.authorized.storage.getSettings()
-                : this.account_settings;
-
-        const readonly_fields = [...get_settings.immutable_fields, ...['immutable_fields', 'email', 'password']];
-        return Object.keys(get_settings).filter(field => !readonly_fields.includes(field));
+        const readonly_fields = [
+            ...(this.account_settings.immutable_fields || []),
```

Comment:
`?? [ ]` is preferred over `|| [ ]` by ESlint, right?

Reply:
thinks so, but we don’t except here something different from array. so I believe '??' is more confusing than helpful here